acceptance/helper.rb

```diff
@@ -335,7 +338,7 @@ def validate_package_version(host)
       expected_version = get_package_version(host)
 
       Beaker::Log.notify "Expecting package version: '#{expected_version}', actual version: '#{installed_version}'"
-      if installed_version != expected_version
+      if installed_version != expected_version and expected_version != 'latest'
         raise RuntimeError, "Installed version '#{installed_version}' did not match expected version '#{expected_version}'"
```

Comment:
The installed version value will never be 'latest'. It will be the concrete version number installed. So, setting expected version to 'latest' will always fail this comparison.

I could move the conditional so that it is easier to read:
```
if expected_version == 'latest'
  # we cannot accurately compare to 'latest'
  return
elsif installed_version != expected_version
  raise RuntimeError, "Installed version '#{installed_version}' did not match expected version '#{expected_version}'"
end
```